docs/podman-restart.1.md

```diff
@@ -0,0 +1,46 @@
+% podman(1) podman-restart - Restart a container
+% Matt Heon
+# podman-restart "1" "March 2017" "podman"
+
+## NAME
+podman restart - Restart a container
+
+## SYNOPSIS
+**podman attach [OPTIONS] CONTAINER [CONTAINER...]**
+
+## DESCRIPTION
+The restart command allows containers to be restarted using their ID or name.
+Containers will be stopped if they are running and then restarted. Stopped
+containers will not be stopped and will only be started.
+
+## OPTIONS
+**--timeout**
+Timeout to wait before forcibly stopping the container
```

Comment:
Should we note the default timeout here?

Reply:
Actually do we have one, or only just in the tests?  I'm not seeing it in libpod.conf. If we don't have one, ignore this.

Reply:
It's hardcoded somewhere. Probably a good idea to make it configurable for the entire runtime at some point...

The timeout isn't in the `podman stop` manpage so I omitted it here